runtime/vm/VMAccess.cpp

```diff
@@ -1010,6 +1024,11 @@ waitForExclusiveVMAccessMetronome(J9VMThread * vmThread, UDATA responsesRequired
 void
 waitForExclusiveVMAccessMetronomeTemp(J9VMThread * vmThread, UDATA vmResponsesRequired, UDATA jniResponsesRequired)
 {
+	J9JavaVM *vm = vmThread->javaVM;
+
+	/* should not try to wait since we should have not requested it in the first place */
+	Assert_VM_true(FALSE == vm->alreadyHaveExclusive);
+
 	waitForResponseFromExternalThread(vmThread->javaVM, vmResponsesRequired, jniResponsesRequired);
```

Comment:
`vm` local variable can also be used here.
```suggestion
	waitForResponseFromExternalThread(vm, vmResponsesRequired, jniResponsesRequired);
```